hypothesis-python/src/hypothesis/internal/conjecture/utils.py

```diff
@@ -284,6 +284,7 @@ def biased_coin(
                 # (see https://bugs.python.org/issue2506), coverage incorrectly
                 # thinks that this condition is always true.  You can trivially
                 # check by adding `else: assert False` and running the tests.
+                # Note: Fixed in both CPython and PyPy 3.10, with PEP 626
```

Comment:
Oh, nice!  Since we now only support Python >= 3.8, and run coverage jobs on 3.10, let's just delete the pragma and this whole comment ☺️